src/transactions/test/OfferTests.cpp

```diff
@@ -674,350 +674,435 @@ TEST_CASE("create offer", "[tx][offers]")
                     {}, [&] { return market.addOffer(a1, offerState); }));
             }
 
-            SECTION("offer does not cross")
+            enum class AuthState
             {
-                for_all_versions(*app, [&] {
-                    issuer.pay(b1, usd, 20000);
-                    // offer is sell 40 USD for 80 IDR ; sell USD @ 2
-                    market.requireChangesWithOffer({}, [&] {
-                        return market.addOffer(b1, {usd, idr, Price{2, 1}, 40});
-                    });
-                });
-            }
+                AUTHORIZED,
+                BUY_AUTHORIZED_TO_MAINTAIN_LIABILITIES,
+                SELL_AUTHORIZED_TO_MAINTAIN_LIABILITIES
+            };
+            auto multipleOffers = [&](AuthState authState) {
+                auto a1CheckSetAuthToMaintainLiabilities = [&]() {
```

Comment:
Nitpick: same thing as other "check" function. Do you mean something like "a1MaybeSet..."?

Reply:
Fixed. I think I meant "checkToSet..." but "maybe" sounds better.